### PR TITLE
UIDEXP-321 Update translations in logs list

### PR DESCRIPTION
**Pull request by vashjs**
## Purpose

https://issues.folio.org/browse/UIDEXP-321
## Pre-Merge Checklist
Before merging this PR, please go through the following list and take appropriate actions.
- [ ] I've added appropriate record to the CHANGELOG.md
- Does this PR meet or exceed the expected quality standards?
  - [ ] Code coverage on new code is 80% or greater
  - [ ] Duplications on new code is 3% or less
  - [ ] There are no major code smells or security issues
- Does this introduce breaking changes?
  - [ ] If any API-related changes - okapi interfaces and permissions are reviewed/changed correspondingly
  - [ ] There are no breaking changes in this PR.

If there are breaking changes, please **STOP** and consider the following:

- What other modules will these changes impact?
- Do JIRAs exist to update the impacted modules?
  - [ ] If not, please create them
  - [ ] Do they contain the appropriate level of detail?  Which endpoints/schemas changed, etc.
  - [ ] Do they have all they appropriate links to blocked/related issues?
- Are the JIRAs under active development?
  - [ ] If not, contact the project's PO and make sure they're aware of the urgency.
- Do PRs exist for these changes?
  - [ ] If so, have they been approved?

Ideally all of the PRs involved in breaking changes would be merged in the same day to avoid breaking the folio-testing environment.  Communication is paramount if that is to be achieved, especially as the number of intermodule and inter-team dependencies increase.

While it's helpful for reviewers to help identify potential problems, ensuring that it's safe to merge is ultimately the responsibility of the PR assignee.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -24,6 +24,7 @@
   },
   "devDependencies": {
     "@babel/eslint-parser": "^7.18.2",
+    "@babel/plugin-proposal-class-properties": "^7.18.6",
```
Comment (BogdanDenis) [inline package.json:27]: Some `-proposal-` plugins were renamed in `stripes-webpack` to `-transform-` because they became standard and will were renamed by babel.
https://github.com/folio-org/stripes-webpack/pull/114
Please rename the same plugins here and in babel.config.js

**Review by BogdanDenis — APPROVED** [review]

**Review by UladzislauKutarkin — APPROVED** [review]

File: package.json [hunk 2]
```diff
@@ -24,6 +24,7 @@
   },
   "devDependencies": {
     "@babel/eslint-parser": "^7.18.2",
+    "@babel/plugin-transform-class-properties": "^7.22.5",
```
Comment (zburke) [inline package.json:27]: These changes belong in a different PR. Also note that instead of maintaining these deps here and syncing them with a custom `babel.config.js`, a simpler/safe option is to just leverage the config used by stripes-webpack and exported via stripes-cli, [e.g. in stripes-core](https://github.com/folio-org/stripes-core/blob/a5e888b9a102d25f471dc5306feba02cc2e2e82b/test/jest/babel.config.js):
```
const { babelOptions } = require('@folio/stripes-cli');

// ... customizations

module.exports = {
  ...babelOptions,
};
```

File: package.json [hunk 3]
```diff
@@ -52,9 +53,9 @@
     "react-intl": "^5.7.0",
     "react-query": "^3.6.0",
     "react-router-dom": "^5.2.0",
+    "react-virtualized-auto-sizer": "*",
     "regenerator-runtime": "^0.13.3",
-    "sinon": "^7.1.1",
-    "react-virtualized-auto-sizer": "*"
+    "sinon": "^7.1.1"
```
Comment (zburke) [inline package.json:58]: I understand this is just yarn reorganizing deps, but it is never save to depend on a `*` version. RVAS, in particular, has been the source of some really sneaky problems over the last few months ([STCOM-1148](https://issues.folio.org/browse/STCOM-1148)). If you notice this kind of thing in the future, please file a ticket to fix it. I filed [UIDEXP-325](https://issues.folio.org/browse/UIDEXP-325) for this one.

**Review by zburke — COMMENTED** [review]
Please use separate PRs for separate tasks. Obviously there is little risk that you'll need to revert a PR that merely changes translations, but the principle still stands.
